src/components/talks/LiveDemo/index.tsx

```diff
@@ -29,22 +25,19 @@ const LiveDemo = ({
       </Link>
     </Caption>
     <Video
-      controls
       style={{
         aspectRatio: `${videoWidth} / ${videoHeight}`,
         maxWidth: videoWidth,
       }}
     >
-      <source src={videoSource} type={videoType} />
-      {subtitlesSource && (
-        <track
-          label="English"
-          kind="subtitles"
-          srcLang="en"
-          src={subtitlesSource}
-          default
-        />
-      )}
+      <iframe
+        width="100%"
+        src={videoSource}
+        title={title}
+        frameBorder={0}
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+        allowFullScreen
+      ></iframe>
```

Comment:
I’d suggest making this iframe lazy (`loading=lazy`) – wdyt? This should delay loading it and boost the Lighthouse scores a bit.

Reply:
And, as mentioned in Telegram, let’s also tune the UX a bit:
- turn off the “More Videos” block (I think this is done by enabling the `rel` param; [see the docs](https://developers.google.com/youtube/player_parameters))
- enable subtitles by default (`cc_load_policy`?)
- reduce YouTube branding (`modestbranding`)